### PR TITLE
majid/portfolio_ws

### PR DESCRIPTION
**Pull request by majid-binary**

### CODE REVIEW

**Comment by raunakkathuria** [comment]
@qingweibinary can we use your flex table function in this?

File: src/javascript/binary/pages/portfiliows.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    /**
+     * Updates portfolio table
+    **/
+    var updatePortfolio = function(data) {
+
+        /**
+         * Check for error
+        **/
+        if("error" in data) {
+            throw new Error("Trying to get portfolio data, we got this error", data.error);
```
Comment (raunakkathuria) [inline src/javascript/binary/pages/portfiliows.js]: How we are handling the localize for these error messages?

**Comment by qingweibinary** [comment]
I believe yes, just that to make the table generic enough, it does not provide much functionality out of the box

File: src/javascript/binary/static/common/string_util.js [hunk 2]
```diff
@@ -31,12 +31,34 @@ var StringUtil = (function(){
         return momentObj.toString();
     }
 
+    /**
+     * Format currency
+     * formatCurrency(20, "USD") -> "USD 20.00"
+     * formatCurrency("10000", "GBP") -> "GBP 10,000.00"
+     * formatCurrency(10.027, "EUR") -> "EUR 10.027"
+    **/
+    function formatCurrency(n, c) {
```
Comment (jy-binary) [inline src/javascript/binary/static/common/string_util.js]: to be removed and use Negar's function instead

File: src/javascript/binary/pages/portfiliows.js (hunk 1 above)
Comment (majid-binary) [inline src/javascript/binary/pages/portfiliows.js]: This is not shown in the interface, just in console. Do we need localization for it?

**Pull request merged**